recipes/dsdp/dsdp_osx_patch.diff

```diff
@@ -0,0 +1,13 @@
+diff -Naur DSDP5.8/src/sys/dsdploginfo.c DSDP5.8_patched/src/sys/dsdploginfo.c
+--- DSDP5.8/src/sys/dsdploginfo.c	2018-04-20 06:40:53.000000000 +0200
++++ DSDP5.8_patched/src/sys/dsdploginfo.c	2018-04-20 06:41:19.000000000 +0200
+@@ -6,7 +6,9 @@
+ #include <stdarg.h>
+ #include <sys/types.h>
+ #include <stdlib.h>
++#ifndef __MACH__
+ #include <malloc.h>
```

Comment:
TBH `malloc.h` shouldn't be used at all here. `stdlib.h` provides all of this functionality and is the POSIX compliant. `malloc.h` is not.

ref: https://stackoverflow.com/a/12973361